test/index.js

```diff
@@ -0,0 +1,61 @@
+const seleniumAssistant = require('selenium-assistant');
```

Comment:
Same feedback here as the other scripts, highly recommend using `esm` so all your code can be module driven.

Reply:
done